composer.json

```diff
@@ -55,10 +55,10 @@
         "php-http/promise": "^1.0",
         "polishsymfonycommunity/symfony-mocker-container": "^1.0",
         "symfony/browser-kit": "^3.4.34 || ^4.2.12 || ^5.0",
-        "symfony/cache": "^3.4.34 || ^4.2.12 || ^5.0",
+        "symfony/cache": "^3.4.34 || ~4.2.12 || ^4.3.8 || ^5.0",
         "symfony/dom-crawler": "^3.4.34 || ^4.2.12 || ^5.0",
         "symfony/framework-bundle": "^3.4.34 || ^4.2.12 || ^5.0",
-        "symfony/http-foundation": "^3.4.34 || ^4.2.12 || ^5.0",
+        "symfony/http-foundation": "^3.4.34 || ~4.2.12 || ^4.3.8 || ^5.0",
```

Comment:
why is the security warning still present with this? did i mess up or does their checker not understand all details of composer version constraints?